.github/workflows/release-internal.yml

```diff
@@ -0,0 +1,64 @@
+name: Release internal patch
+
+on:
+  workflow_dispatch:
+    inputs:
+      version_number:
+        description: "The release version number (i.e. 1.0.0b1)"
+        type: string
+        required: true
+      sha:
+        description: "The sha to use (leave empty to use latest on main)"
```

Comment:
I would make this a required field and default it to `main` if the expectation is that we will mostly run this off `main`.  It makes manually submissions more explicit.  Maybe rename this input to "sha_or_main" to make it really clear in the dropdown.

Reply:
Why would we have a field that's both required and has a default? Wouldn't the default never be used?

Reply:
The default should be in text form upon dispatch, so it's a default argument, not a default value if nothing given.

Reply:
Exactly.  Make it required so we're explicit.  But give it the default if we expect to be using main the majority of the time.